recipes/zeus_expected/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.cmake import cmake_layout
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+from conan.tools.files import apply_conandata_patches, export_conandata_patches
+import os
+
+required_conan_version = ">=1.54.0"
+
+
+class ZeusExpectedConan(ConanFile):
+    name = "zeus_expected"
+
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/zeus-cpp/expected"
+    description = "Backporting std::expected to C++17."
+    topics = ("cpp17", "expected")
+    license = "MIT"
+
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    # patches applied
+    # no_copy_source = True
```

Comment:
```suggestion
    no_copy_source = True
```

As hearder only project and you are actually only copying file, does not worth keeping a build folder, it would be a copy of a copy. So we can copy from source directly. The patch will be applied directly to source, but that's a common usage.

Reply:
Thanks for explanation. I misunderstood it